prototype2/client/src/engine/control/menu.rs

```diff
@@ -0,0 +1,46 @@
+use specs;
+use engine;
+
+pub struct System;
+
+pub struct MenuState {
+  open: bool,
+}
```

Comment:
i know the menu isnt actually a thing yet, but seeing this thing called MenuState as not an enum feels Wrong